apps/src/music/player/MusicLibrary.ts

```diff
@@ -25,6 +25,17 @@ export default class MusicLibrary {
     }
   }
 
+  getDefaultSound(): string | undefined {
+    const firstGroup: FolderGroup = this.groups[0];
+
+    // The fallback is the first non-instrument/kit folder's first sound.
+    const firstFolder = firstGroup?.folders.find(group => !group.type);
+    const firstSoundFullPath = `${firstFolder?.path}/${firstFolder?.sounds[0].src}`;
+
+    // Return the specified default sound or the fallback sound.
+    return firstGroup?.defaultSound || firstSoundFullPath;
```

Comment:
minor: early return `firstGroup.defaultSound` if it exists, before searching for firstFolder?

Reply:
Done.